<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.1.1 (Python 3.13.1 on win32)" generated="2025-03-17T19:37:31.180414" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="c:\Users\smibc\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-03-17T19:37:31.297530" elapsed="10.330920"/>
</kw>
<status status="PASS" start="2025-03-17T19:37:31.295721" elapsed="10.333157"/>
</test>
<status status="PASS" start="2025-03-17T19:37:31.183984" elapsed="10.446586"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" skip="0" id="s1" name="Robot Interactive Console">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-03-17T19:37:31.166860" level="WARN">Error in file 'c:\Users\smibc\.vscode\extensions\robocorp.robotframework-lsp-1.13.0\src\robotframework_ls\vendored\robotframework_interactive\robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
